client/gutenberg/extensions/slideshow/view.js

```diff
@@ -0,0 +1,23 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+
+import './style.scss';
+import component from './component.js';
+import { settings } from './settings.js';
+import FrontendManagement from 'gutenberg/extensions/shared/frontend-management.js';
+
+window &&
```

Comment:
To protect against missing `window` well need to 
```suggestion
typeof window !== 'undefined' &&
```

or something similar. This will still error outside of browser context I believe.

Reply:
If correct, this would be a problem in Map as well. Besides mobile, what other non-browser contexts are there? And how would I go about verifying this fix in a non-browser context?

Reply:
Unguarded `window` usage is often a pain in tests and server side rendering. You can verify that this will error as follows:

```
$ node -e 'window && console.log( "No problem, the window check failed" );'

[eval]:1
window && console.log( "No problem, the window check failed" );
^

ReferenceError: window is not defined
    at [eval]:1:1
…
```

Reply:
Makes sense. Since I have an identical problem in the Map block, would it be OK with you if I leave this in, and then after we land this PR open a new one that fixes the unguarded `window` in Slideshow & Map simultaneously?

Reply:
At @scruffian's suggestion, adding the window check here just to be safe. I'll separately add to Map block in a different PR. Change made in https://github.com/Automattic/wp-calypso/pull/29419/commits/0d043045480e06d5204f61cdbe6e06881498b084